src/route/lwtunnel.rs

```diff
@@ -106,31 +126,160 @@ impl std::fmt::Display for RouteLwEnCapType {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy, Default)]
```

Comment:
Please add `#[non_exhaustive]` so we don't break API when adding new entry to this enum.